kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -4,21 +4,21 @@ set -e
 function collect_deployment_and_operations {
     DEPLOYLOGSDIR=/var/log/azure/arm-deployments/
     sudo mkdir -p /var/log/azure/arm-deployments/
-
+    
```

Comment:
could you please use update the customData in template?

Reply:
I will have a lot more changes for this file in the next few days. 
Probably fine as we are not releasing anytime soon?
I am hoping to automate the process

Reply:
Ok.